react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom/client';
+
+const buttonStyle = {
+  color: 'blue'
+};
+
+function HotButton() {
+  const [index, setIndex] = useState(0);
+  function handleClick() {
+    const newIndex = index + 1;
+    setIndex(newIndex);
+    if (newIndex < 3) {
+      buttonStyle.color = 'blue';
```

Comment:
Since you only use `buttonStyle.color` in this function, you could just use a string instead of an object.